RDSExpoApp/app/(tabs)/_layout.tsx

```
@@ -0,0 +1,35 @@
+import FontAwesome from '@expo/vector-icons/FontAwesome';
+import { Tabs } from 'expo-router';
+import React from 'react';
+
+const colorScheme: 'light' | 'dark' = 'light';
+
+export default function TabLayout() {
+  return (
+    <Tabs
+      screenOptions={{
+        tabBarActiveTintColor: Colors[colorScheme].tint,
+        headerShown: false, 
```

ESLint

[error] Delete `·`

(prettier/prettier)